src/utils/emailPattern.js

```diff
@@ -0,0 +1 @@
+export const emailPattern = /^[\w.+-]+@([\w-]+\.){1,3}[\w-]{2,}$/;
```

Comment:
Add [empty line in the end of file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)